modules/UI/util/MessageHandler.js

```diff
@@ -117,7 +117,7 @@ const messageHandler = {
             cls,
             messageKey,
             messageArguments,
-            timeout = NOTIFICATION_TIMEOUT) {
+            timeout = NOTIFICATION_TIMEOUT_TYPE.SHORT) {
```

Comment:
Can you please go ahead and remove this `participantNotification` altogether? It's only used by UI.js which is then called by conference.js, so we should be able to remove all that legacy code and dispatch the notifications directly from conference.js